lib/min_heap.rb

```diff
@@ -8,27 +8,32 @@ def initialize(key, value)
 end
 
 class MinHeap
-
   def initialize
     @store = []
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(log n)
+  # Space Complexity: constant
```

Comment:
Since `heap_up` is going recursively, you will have some space complexity for the system stack.